zeppelin-web/add_dependencies.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+( sudo apt-get install libfontconfig -y || sudo yum install fontconfig -y ) || exit 0;
```

Comment:
To be honest this is not the best solution. The best solution would be present all dependencies in phantomjs for all planforms. But this workaround just work in linux and provide automatical build with maven and not required from e.g. (backend developers and wondows users) to be familiar with hodejs npm yarn and other UI-specific things like hidden dependencies and so on.